app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala

```diff
@@ -53,6 +54,24 @@ object FormTemplateValidator {
     }
   }
 
+  def validateRepeatingSectionFields(sectionList: List[Section]): ValidationResult = {
+    sectionList.map { section =>
+      (section.repeatsMax, section.repeatsMin) match {
+        case (Some(repMax), Some(repMin)) if !repMax.equals(repMin) =>
```

Comment:
This condition caught my eye. Why the only valid combination is repMax the same as repMin ?